docs/epic/EPIC-v3.0-Synthesis-Architecture.md

```
@@ -330,60 +324,58 @@ Based on [opencode.ai/docs](https://opencode.ai/docs/) and GitHub research:
 ```
 
-**Production Agents with Tiers:**
-| Agent | Quick | Standard | Advanced |
-|-------|-------|----------|----------|
-| **Engineer** | Qwen3 Coder | Kimi K2.5 | GPT-5.3 Codex |
-| **Architect** | Kimi K2.5 | Kimi K2.5 | Claude Sonnet 4.6 |
-| **Pentester** | Qwen3 Coder | Kimi K2.5 | Claude Sonnet 4.6 |
-| **Researcher** | Kimi K2 | Kimi K2.5 | Claude Sonnet 4.6 |
-| **Designer** | Gemini 3 Flash | Gemini 3 Flash | Gemini 3 Pro |
+**Production Agent Models:**
+| Agent | Configured Model | Use Case |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)